Possible lastfm build failure

Seen on one of my older Gentoo systems:

```
FAILED: gerbera 
: && /usr/bin/x86_64-pc-linux-gnu-g++ -march=amdfam10 -O2 -pipe -Wl,-O1 -Wl,--as-needed CMakeFiles/gerbera.dir/src/main.cc.o -o gerbera  liblibgerbera.a  -latomic  -lc  -luuid  /usr/lib64/libupnp.so  /usr/lib64/libixml.so  /usr/lib64/libspdlog.so.1.8.2  -lpthread  /usr/lib64/libfmt.so.7.1.2  -Wl,--as-needed  /usr/lib64/libpugixml.so  /usr/lib64/libsqlite3.so  /usr/lib64/libnsl.so  /usr/lib64/libduktape.so  -L/usr/lib64 -ltag -lz  /usr/lib64/libavformat.so  /usr/lib64/libavutil.so  -lcurl  /usr/lib64/libmagic.so  /usr/lib64/libffmpegthumbnailer.so  /usr/lib64/libexif.so  /usr/lib64/libebml.so  /usr/lib64/libmatroska.so  -llastfmlib  -lcurl  /usr/lib64/libmagic.so  /usr/lib64/libffmpegthumbnailer.so  /usr/lib64/libexif.so  /usr/lib64/libebml.so  /usr/lib64/libmatroska.so  -llastfmlib  -lpthread  /usr/lib64/libz.so && :
/usr/lib/gcc/x86_64-pc-linux-gnu/9.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: liblibgerbera.a(lastfm_scrobbler.cc.o): in function `LastFm::startedPlaying(std::shared_ptr<CdsItem>)':
lastfm_scrobbler.cc:(.text+0x747): undefined reference to `HMSFToMilliseconds(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
collect2: error: ld returned 1 exit status
```

--- a/src/util/tools.h
+++ b/src/util/tools.h
@@ -189,7 +189,7 @@ std::string getProtocol(const std::string& protocolInfo);
 std::string millisecondsToHMSF(int milliseconds);
 
 /// \brief converts a "H*:MM:SS.F*" representation to milliseconds
-int HMSFToMilliseconds(const std::string& time);
+int HMSFToMilliseconds(std::string_view time);
 
 /// \brief Extracts resolution from a JPEG image
 std::string get_jpeg_resolution(const std::unique_ptr<IOHandler>& ioh);

